2024-07-28 16:23:33,820 INFO    Thread-21 (_run_job):11812 [wandb_setup.py:_flush():76] Current SDK version is 0.17.4
2024-07-28 16:23:33,820 INFO    Thread-21 (_run_job):11812 [wandb_setup.py:_flush():76] Configure stats pid to 11812
2024-07-28 16:23:33,820 INFO    Thread-21 (_run_job):11812 [wandb_setup.py:_flush():76] Loading settings from C:\Users\ge84gac\.config\wandb\settings
2024-07-28 16:23:33,820 INFO    Thread-21 (_run_job):11812 [wandb_setup.py:_flush():76] Loading settings from C:\Users\ge84gac\Gate_qbm\tests\wandb\settings
2024-07-28 16:23:33,820 INFO    Thread-21 (_run_job):11812 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'entity': 'shivang_arora', 'project': 'Gate-qbm-anomaly-detection-transverse', 'root_dir': 'C:\\Users\\ge84gac\\Gate_qbm\\tests', 'run_id': '7pq7dx3e', 'sweep_param_path': 'C:\\Users\\ge84gac\\Gate_qbm\\tests\\wandb\\sweep-iok1td1z\\config-7pq7dx3e.yaml', 'sweep_id': 'iok1td1z'}
2024-07-28 16:23:33,825 INFO    Thread-21 (_run_job):11812 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-07-28 16:23:33,826 INFO    Thread-21 (_run_job):11812 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-07-28 16:23:33,826 INFO    Thread-21 (_run_job):11812 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-07-28 16:23:33,826 INFO    Thread-21 (_run_job):11812 [wandb_init.py:_log_setup():529] Logging user logs to C:\Users\ge84gac\Gate_qbm\tests\wandb\run-20240728_162333-7pq7dx3e\logs\debug.log
2024-07-28 16:23:33,826 INFO    Thread-21 (_run_job):11812 [wandb_init.py:_log_setup():530] Logging internal logs to C:\Users\ge84gac\Gate_qbm\tests\wandb\run-20240728_162333-7pq7dx3e\logs\debug-internal.log
2024-07-28 16:23:33,826 INFO    Thread-21 (_run_job):11812 [wandb_init.py:_jupyter_setup():475] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001FDC8382490>
2024-07-28 16:23:33,826 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():569] calling init triggers
2024-07-28 16:23:33,827 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():576] wandb.init called with sweep_config: {'batchsize': 73, 'beta_eff': 1.7782274212954747, 'epochs': 7, 'hnodes': 1, 'learning_rate': 0.6058642443189942, 't_bias': -0.937237930639048}
config: {}
2024-07-28 16:23:33,827 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():619] starting backend
2024-07-28 16:23:33,827 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():623] setting up manager
2024-07-28 16:23:33,837 INFO    Thread-21 (_run_job):11812 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-07-28 16:23:33,837 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():631] backend started and connected
2024-07-28 16:23:33,864 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_config_callback():1382] config_cb None None {'batchsize': 73, 'beta_eff': 1.7782274212954747, 'epochs': 7, 'hnodes': 1, 'learning_rate': 0.6058642443189942, 't_bias': -0.937237930639048}
2024-07-28 16:23:33,865 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_label_probe_notebook():1334] probe notebook
2024-07-28 16:23:33,865 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_label_probe_notebook():1344] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-07-28 16:23:33,865 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():720] updated telemetry
2024-07-28 16:23:33,966 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-07-28 16:23:34,384 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_on_init():2402] communicating current version
2024-07-28 16:23:34,514 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_on_init():2411] got version response upgrade_message: "wandb version 0.17.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-07-28 16:23:34,514 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():804] starting run threads in backend
2024-07-28 16:23:42,077 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_console_start():2380] atexit reg
2024-07-28 16:23:42,077 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2024-07-28 16:23:42,077 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_redirect():2300] Wrapping output streams.
2024-07-28 16:23:42,077 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_redirect():2325] Redirects installed.
2024-07-28 16:23:42,078 INFO    Thread-21 (_run_job):11812 [wandb_init.py:init():847] run started, returning control to user process
2024-07-28 16:23:42,079 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_config_callback():1382] config_cb None None {'restricted': 'false'}
2024-07-28 16:41:48,404 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_finish():2109] finishing run shivang_arora/Gate-qbm-anomaly-detection-transverse/7pq7dx3e
2024-07-28 16:41:48,404 INFO    Thread-21 (_run_job):11812 [jupyter.py:save_history():444] not saving jupyter history
2024-07-28 16:41:48,404 INFO    Thread-21 (_run_job):11812 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-07-28 16:41:48,404 INFO    Thread-21 (_run_job):11812 [wandb_init.py:_jupyter_teardown():457] cleaning up jupyter logic
2024-07-28 16:41:48,404 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_atexit_cleanup():2349] got exitcode: 0
2024-07-28 16:41:48,404 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_restore():2332] restore
2024-07-28 16:41:48,404 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_restore():2338] restore done
2024-07-28 16:41:57,255 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_footer_history_summary_info():3991] rendering history
2024-07-28 16:41:57,255 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_footer_history_summary_info():4023] rendering summary
2024-07-28 16:41:57,271 INFO    Thread-21 (_run_job):11812 [wandb_run.py:_footer_sync_info():3950] logging synced files
